src/config/mod.rs

```diff
@@ -35,12 +36,12 @@ fn get_config_file_name() -> String {
 /// ```
 /// assert!(check_config_exists());
 /// ```
-fn check_config_exists() -> bool {
-    fs::metadata(get_config_file_name()).is_ok()
+fn check_config_exists() -> Result<bool> {
+    Ok(fs::metadata(get_config_file_name()).is_ok())
```

Comment:
Why is this necessary if Ok is returned always?

Reply:
It's just to propagate the error from create config all the way to main, so it doesn't panic 

Also, in the future there's value in adding a proper cli library like clap